R/ggridges.R

```diff
@@ -0,0 +1,289 @@
+#' Get data for ridge plots
+#'
+#' @param data dataframe, the data returned by `ggplot2::ggplot_build()`.
+#' @param na.rm boolean, from params
+#'
+#' @return dataframe containing plotting data 
+#'
+get_ridge_data <- function(data, na.rm) {
+    if (isTRUE(na.rm)) {
```

Comment:
I'd prefer to stick with 2 space indentation (to match the rest of the code base)

```suggestion
  if (isTRUE(na.rm)) {
```

Reply:
done